### PR TITLE
Docs improvements - Added February changelog, fixed typos, updated links and removed duplicate files

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
- Added february chnagelog.
- Fixed typos and grammar issues in the changelog files
- Updated links for TMS 
- Updated roadmap to remove orchestra and use chainhooks
- Updated docusaurus config and sidebar files.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -425,7 +425,11 @@ module.exports = {
             title: 'Hiro Developers',
             items: [
               {
-                label: 'Stacks API',
+                label: 'Documentation',
```
Comment (sabbyanandan) [inline docusaurus.config.js:428]: Is this a redirect to the same docs site? Curious why.

File: docs/references/changelog-december.md [hunk 2]
```diff
@@ -4,16 +4,16 @@ title: December 2022 Product Changelog
 
 ## Overview
```
Comment (sabbyanandan) [inline docs/references/changelog-december.md:5]: nit: to reduce clutter, might be helpful to move all these into a new "changelog" directory.

File: sidebars.js [hunk 3]
```diff
@@ -269,6 +269,7 @@ module.exports = {
         'references/changelog-november',
         'references/changelog-december',
         'references/changelog-january',
+        'references/changelog-february',
```
Comment (sabbyanandan) [inline sidebars.js:272]: To make it easier for the readers, we could sort this in descending order, so the latest always shows up first. That can avoid scrolling in the future when there are dozens of these links.

**Review by sabbyanandan — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 3 above)
Comment (LakshmiLavanyaKasturi) [reply sidebars.js:272]: Make sense. Will do it.

File: docusaurus.config.js (hunk 1 above)
Comment (LakshmiLavanyaKasturi) [reply docusaurus.config.js:428]: It was initially directed to docs.hiro.so/intro, which no longer exists. A broken link on the current page. 

We can completely remove this link or direct it to our landing page. Please let me know your thoughts.

Comment (LakshmiLavanyaKasturi) [reply docusaurus.config.js:428]: I just realized the Documentation link is only showing in my local build. I can update my build. 

image

Comment (sabbyanandan) [reply docusaurus.config.js:428]: Yes, lets please remove it. Redundant.

File: sidebars.js [hunk 4]
```diff
@@ -256,6 +243,19 @@ module.exports = {
         },
       ],
     },
+    {
+      type: 'category',
+      label: 'Token metadata API',
```
Comment (sabbyanandan) [inline sidebars.js:248]:
```suggestion
      label: 'Token Metadata API',
```

**Review by sabbyanandan — APPROVED** [review]
Let's go!

**Pull request merged**